anesthetic/tension.py

```diff
@@ -0,0 +1,90 @@
+"""Tension statistics between two datasets."""
+from anesthetic.samples import Samples
+from scipy.stats import chi2
+
+
+def tension_stats(A, B, AB, nsamples=None, beta=None):  # noqa: D301
+    """Compute tension statistics between two samples.
+
+    Using nested sampling we can compute:
+
+    - ``logZ``: Bayesian evidence
+
+      .. math::
+          \\log Z = \\log \\int L \\pi d\\theta
+
+    - ``D_KL``: Kullback--Leibler divergence
+
+      .. math::
+          D_\\mathrm{KL} = \\int P \\log(P / \\pi) d\\theta
+
+    - ``logL_P``: posterior averaged log-likelihood
+
+      .. math::
+          \\langle\\log L\\rangle_P = \\int P \\log L d\\theta
+
+    - ``d_G``: Gaussian model dimensionality
+      (or posterior variance of the log-likelihood)
+
+      .. math::
+          d_\\mathrm{G}/2 = \\mathrm{var}(\\log L)_P
+
+    - ``p``: p-value for the tension between two samples
+
+      .. math::
+          p = \\int_{d_\\mathrm{G} - 2 \\log S}^{\\infty} \\chi^2 (x)dx
```

Comment:
This was probably copied from `NestedSamples.stats`, let's make this more relevant for the tension module by focusing on `logR`, `logI`, `logS`.

Reply:
Updated! Thanks.

Reply:
I think it's "Logarithm" rather than "Logarithmic", right?